governance/default_test.go

```diff
@@ -240,6 +240,8 @@ func TestGovernance_ClearVotes(t *testing.T) {
 		if ret != val.e {
 			t.Errorf("Want %v, got %v for %v and %v", val.e, ret, val.k, val.v)
 		}
+		avt := gov.adjustValueType(val.k, val.v)
+		gov.RemoveVote(val.k, avt, 0)
```

Comment:
Why is `RemoveVote` added to test?

Reply:
I'm curious too

Reply:
@ehnuje @hqjang-pepper The original test include `voteMap.Clear()` method which clears all votes whether or not the votes are casted. This PR updates the method where only casted votes are cleared. So, `RemoveVote` is the method that sets the vote casted. That is why I added it.

Reply:
oh I see, it makes sense.